### DISCUSSION TITLE
Custom Ner with spacy-transformers Got [E893] on init step

### DISCUSSION THREAD

**Discussion by UniKloMS** [explosion/spaCy#12515] [Help: Installation]
Hi there, I am new to SpaCy. I am trying to train a vanilla NER model with Spacy Transformers pipelines.
When creating config file, I got "[E893] Could not find function 'spacy-transformers.BARTTokenizer.v1' in function registry 'tokenizers'"
Does anyone know how to resolve it?

FYI: I have installed the following Spacy packages:
spacy                              3.5.1
spacy-alignments           0.9.0
spacy-legacy                  3.0.12
spacy-loggers                1.0.4
spacy-transformers        1.2.2.

**Comment by svlandeg** [comment]
Hi! Can you elaborate on how you created this config file, are you following some sort of code example you found online?
